packages/crypto/src/secp256k1/keypair.ts

```diff
@@ -43,7 +42,7 @@ export class Secp256k1Keypair implements Keypair {
   }
 
   publicKeyStr(encoding: SupportedEncodings = 'base64pad'): string {
-    return uint8arrays.toString(this.publicKey, encoding)
+    return toString(this.publicKey, encoding)
```

Comment:
is there a reason to import this as just `toString`? if so I think it'd be nice to import it as something like `ui8ToString` or similar. otherwise `toString` is pretty generic & gotta check imports to see what it's coming from